ci-operator/jobs/kube-reporting/metering-operator/kube-reporting-metering-operator-master-presubmits.yaml

```diff
@@ -177,6 +177,51 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )unit,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: api.ci
+    context: ci/prow/vendor
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
I'm not sure I entirely understand rehearsal, but it seems like make vendor is downloading a bunch of files and is hoped to always work, so no need to check it before kicking of rest of the build steps?

Reply:
This is all generated code. The only thing I changed was the ci-operator/config/* files that added this test.

Reply:
This is also more of a verification check that we don't cover anywhere in the repository. I've seen a couple of teams do this kind of check.

Reply:
roger